### ISSUE TITLE
scvi: enable KL annealing

### ISSUE THREAD

**Issue by sjfleming** [cellarium-ai/cellarium-ml#316]
Enable a cellarium model (in a way that plays nicely with lightning) to anneal some prefactor in a loss function -- in this case, a prefactor for the KL divergence in the loss.

It should be able to follow some kind of a schedule during training (like a linear ramp-up to 1 in our case, over a defined number of epochs).

Figure out the RIGHT way to do this in lightning.  Look for examples.

**Comment by sjfleming** [comment]
Looks like scvi-tools implements it this way
https://github.com/scverse/scvi-tools/blob/557118bb03d1750609a10e0876c171d5b45a21eb/src/scvi/train/_trainingplans.py#L37-L78

**Comment by sjfleming** [comment]
Looks like these are the default training args for scvi-tools run of scvi
https://github.com/scverse/scvi-tools/blob/557118bb03d1750609a10e0876c171d5b45a21eb/src/scvi/train/_trainingplans.py#L148-L165

**Comment by sjfleming** [comment]
Closed by #328
